app/helpers/films_helper.rb

```diff
@@ -0,0 +1,11 @@
+require 'csv'
+
+module FilmsHelper
+  def get_csv_data
+    data = []
+    CSV.foreach("lib/assets/netflix_titles.csv", headers: true) do |row|
```

Comment:
you could turn this into 1 liner:
`CSV.foreach("lib/assets/netflix_titles.csv", headers: true).map(&:to_h)`